documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,84 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers Container.
+Currently `systemd` is the default init process in the Clear Containers mini-os
+system. The Systemd project provides a tool called `systemd-bootchart` that
+collects the CPU load, disk load, memory usage, as well as per-process information
+from the os boot process.
+
+## Configure Clear Containers to get a bootchart ##
+
+To get a bootchart from a Clear Container, follow the steps below.
+
+### Prepare your Clear Container image ###
+
+The file `data/cc-bootchart.conf` can be used get a bootchart graph.
+
+The file must be installed in a Clear Container image with `systemd` and `systemd-bootchart`.
+
+The file data/cc-bootchart can be installed with the configure option
+--with-cc-image-bootchart-config-dir=DIR DIR where DIR is any of the following paths.
```

Comment:
Emphasize the options. Please explicitly state the commands with the appropriate paths.